Buddies.API/Controllers/ProjectsController.cs

```diff
@@ -243,7 +245,209 @@ public async Task<ActionResult<SearchResponse>> GetUser(string search, int page,
             return Ok(response);
         }
 
-             
+        /// <summary>
+        /// API route GET /api/v1/projects/:id for fetching project profile.
+        /// </summary>
+        [HttpGet("{id}")]
+        public async Task<ActionResult> GetProfile(int id)
+        {
+            var project = await _context.Projects.FindAsync(id);
+            if (project == null)
+            {
+                return NotFound("PROJECT PROFILE NOT FOUND");
+            }
+            var owner = await _context.Profiles.FindAsync(project.OwnerId);
+            if (owner == null)
+            {
+                return NotFound("OWNER OF PROJECT PROFILE NOT FOUND");
+            }
+            var ownerUser = await _context.Users.FindAsync(project.OwnerId);
+            if (ownerUser == null)
+            {
+                return NotFound("OWNER OF PROJECT PROFILE NOT FOUND");
+            }
+            var profileResponse = new ProjectProfileResponse{
+                Title = project.Title,
+                Description = project.Description,
+                Location = project.Location,
+                Username = owner.FirstName + " " + owner.LastName,
+                Email = ownerUser.Email,
+                Category = project.Category,
+                MaxMembers = project.MaxMembers,
+            };
+
+            var membersInProject = _context.Projects.Where(p => p.ProjectId == id).SelectMany(p => p.Members).ToList();
```

Comment:
Doesn't Project have a [list of members](https://github.com/GDSCUTM-CommunityProjects/Buddies/blob/048063f57e17a951c99dad49bb836e924f7bbd2d/Buddies.API/Entities/Project.cs#L62), do you need to query it again?